src/themes/dark.scss

```diff
@@ -42,6 +42,7 @@
   --wxc-icon--color: #{$wxc-white-alpha-100};
   --wxc-icon--active-color: #{$wxc-green-50};
   --wxc-icon--muted-color: #{$wxc-red-50};
+  --wxc-icon--accent-color: #{$wxc-green-40};
```

Comment:
"active" should be renamed to "accent".

commit: fix(Icons): use correct color for accents